7.5/create-servicebus.ps1

```diff
@@ -0,0 +1,44 @@
+# This script needs the Azue CLI tool installed.
+# Get it from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+
+
+# Enter the name of the subscription. You can use the subscription of your Azure VMs.
+[string] $subscription = "DocuWare Cloud Test"
```

Comment:
How about adding a param section - so there is no need to edit the script file by customers?
It could be kept with default or sample values. So that the script works also without parameters.

Reply:
Good idea, I will do it.